hbase-client/src/main/java/org/apache/hadoop/hbase/client/ClientServiceCallable.java

```diff
@@ -35,8 +37,10 @@ public abstract class ClientServiceCallable<T>
   extends RegionServerCallable<T, ClientProtos.ClientService.BlockingInterface> {
 
   public ClientServiceCallable(Connection connection, TableName tableName, byte[] row,
-    RpcController rpcController, int priority) {
-    super(connection, tableName, row, rpcController, priority);
+    RpcController rpcController, int priority, Map<String, byte[]> requestAttributes) {
+    super(connection, tableName, row,
+      HBaseRpcControllerImpl.configureRequestAttributes(rpcController, requestAttributes), priority,
```

Comment:
oh. maybe its succeeding because of this. we should remove this in favor of my other comment